cmd/make-bm-worker/main.go

```diff
@@ -14,6 +14,7 @@ func main() {
 	var password = flag.String("password", "", "password for BMC")
 	var bmcAddress = flag.String("address", "", "address URL for BMC")
 	var hardwareProfile = flag.String("hardwareprofile", "", "hardwareProfile to be used")
+	var macAddress = flag.String("bootmacaddress", "", "bootMACAddress to be used")
```

Comment:
There's another version of this tool in dev-scripts, which includes "-boot-mac".  Can we use the same name to match?

Reply:
Done.